### DISCUSSION TITLE
Release 0.5.13

### DISCUSSION THREAD

**Discussion by cthackers** [cthackers/adm-zip#492] [General]
@5saviahv Are we ready for the next release ? Want to do any more stuff ?

**Comment by 5saviahv** [comment]
I will add test for #438 and make merge

**Reply by 5saviahv** [reply]
all tests are ok ... so yeah

**Comment by cthackers** [comment]
done

**Comment by cthackers** [comment]
can we put together some release notes ?

**Comment by cthackers** [comment]
Done, release created and npm published. Thank you @5saviahv . Great work!
